tests/config/test_server.py

```diff
@@ -35,7 +35,7 @@ def test_is_threepid_reserved(self):
     def test_unsecure_listener_no_listeners_open_private_ports_false(self):
         conf = yaml.safe_load(
             ServerConfig().generate_config_section(
-                "che.org", "/data_dir_path", False, None
+                "che.org", "/data_dir_path", False, None, "CONFDIR"
```

Comment:
the params to `generate_config_section` are supposed to be passed in as `kwargs`. We should do the same here:

```suggestion
                "che.org", "/data_dir_path", False, None, config_dir_path="CONFDIR"
```

(likewise below)